src/main/java/io/frictionlessdata/datapackage/Dialect.java

```diff
@@ -151,36 +159,36 @@ public static Dialect fromCsvFormat(CSVFormat format) {
     /**
      * Create a new Dialect object from a JSON representation
      * @param json JSON as String representation, eg. from Resource definition
-     * @return new Dialect object with values from JSONObject
+     * @return new Dialect object with values from JsonNode
      */
     public static Dialect fromJson(String json) {
```

Comment:
The `Dialect` object should not be a wrapper for a JSON object: it should have individual fields corresponding to its attributes, and simply use Jackson (de)serialization for the `fromJson` and `getJson` methods.